app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+def copy_file(command: str) -> None:
+    todolist = command.split(" ")
```

Comment:
space is default parameter for split
Also validate that command.split() have 3 parameters